### PR TITLE
Updating file names and creating housing doc md

### PR DESCRIPTION
**Pull request by annikaLiving**
Renamed
- schemas
- autogen docs

Created housing doc

Updated readme

### CODE REVIEW

File: schemas/README.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # Schemas
 
-This directory contains JSON Schema Definitions and JSON Documents relating to these definitions. Information on each subdirectory is as follows:
+This directorys contains JSON sSchema definitions and JSON samples relating to these definitions.
```
Comment (JTA2020) [inline schemas/README.md]: Is there a spelling error here?

Comment (JTA2020) [reply schemas/README.md]: Does not LGTM 👎

**Review by JTA2020 — CHANGES_REQUESTED** [review]
Please review comments and make changes, thanks

File: schemas/README.md (hunk 1 above)
Comment (annikaLiving) [reply schemas/README.md]: Updated the typos

**Review by JTA2020 — APPROVED** [review]
Thank you for these changes, I approve

**Pull request merged**